### PR TITLE
Feature/openapi hardening

### PR DESCRIPTION
**Pull request by dlamoris**

### CODE REVIEW

File: example/openapi/api-docs.yaml [hunk 1]
```diff
@@ -1035,331 +1361,458 @@ components:
...
+          readOnly: true
+        _commitId:
           type: string
-        _created:
+          readOnly: true
+      additionalProperties: true
+    CommitUpdatedJson:
+      type: object
+      properties:
+        previousDocId:
```
Comment (ivan-gomes) [inline example/openapi/api-docs.yaml]: `previousDocId` doesn't start with an underscore but `_docId` does. Also applies to `CommitDeletedJson#previousDocId`.

File: example/openapi/api-docs.yaml [hunk 2]
```diff
@@ -1035,331 +1361,458 @@ components:
...
           type: string
-        commits:
+          nullable: true
+        refs:
           type: array
           items:
-            $ref: '#/components/schemas/CommitJson'
-    ProjectJson:
+            $ref: '#/components/schemas/RefJson'
+    CommitAddedJson:
```
Comment (ivan-gomes) [inline example/openapi/api-docs.yaml]: Can we merge the `Commit + (Added|Updated|Deleted) + Json` models? None of the properties are specified as required, so CommitUpdatedJson (renamed) can be used for all of them.

**Review by ivan-gomes — CHANGES_REQUESTED** [review]

File: example/openapi/api-docs.yaml (hunk 1 above)
Comment (dlamoris) [reply example/openapi/api-docs.yaml]: _ added

File: example/openapi/api-docs.yaml (hunk 2 above)
Comment (dlamoris) [reply example/openapi/api-docs.yaml]: changed to ElementUpdated

File: example/openapi/api-docs.yaml [hunk 3]
```diff
@@ -1035,331 +1412,419 @@ components:
...
+    CommitsRequest:
+      required:
+      - commits
+      type: object
+      properties:
+        commits:
+          type: array
+          items:
+            $ref: '#/components/schemas/Commit'
+    ElementUpdated:
```
Comment (ivan-gomes) [inline example/openapi/api-docs.yaml]: I think `ElementVersion` (consistent with SysML v2), `ElementChange`, `ElementRevision`, etc. may be a better name as update is usually connoted as disjoint from create/delete (CRUD).

**Review by ivan-gomes — CHANGES_REQUESTED** [review]

File: example/openapi/api-docs.yaml (hunk 3 above)
Comment (dlamoris) [reply example/openapi/api-docs.yaml]: changed to ElementVersion

**Pull request merged**